mesheryctl/internal/cli/root/version.go

```diff
@@ -25,6 +25,8 @@ import (
 	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
 	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/constants"
 	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+	log "github.com/sirupsen/logrus"
```

Comment:
No no no no no. Not this @gr455 :rofl: 
We're on a movement of removing directly using logrus in the meshery ecosystem. We have to use utils.Log here in mesheryctl which uses logger from meshkit

Reply:
Oh well, then we also have to change all these errors to meshkit erros